dts/bindings/wifi/nordic,nrf70-coex.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2022 Nordic Semiconductor ASA
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
```

Comment:
Of course, missed this for few files, will fix. Wonder why the `license scan` CI check didn't catch this?

Reply:
That's because it is "allowed" to use other open source licenses than apache, it just has to go through TSC, so for now CI just checks that an identifier is there :)